main.py

```diff
@@ -1,14 +1,18 @@
-import requests
-bestMovies =[]
-for i in range(1,26):
+from requests import Session
+
+sess = Session()
+
+best_movies = []
+for i in range(26):
     params = {
         "page" : i
     }
     #print(params)
-    response = requests.get("http://moviesapi.ir/api/v1/movies",params = params)
-    for i in range(0,10):
-        print(response.json()["data"][i]['title'])
-        bestMovies.append(response.json()["data"][i]['title'])
+    response = sess.get("http://moviesapi.ir/api/v1/movies", params = params)
+    data = response.json()

```

Comment:
http responses are usually a simple text. converting to JSON will take effort, so, ..., just do it once.